includes/sanitizers/require-sanitizers.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+require_once( AMP__DIR__ . '/includes/sanitizers/class-amp-style-sanitizer.php' );
+require_once ( AMP__DIR__ . '/includes/sanitizers/class-amp-blacklist-sanitizer.php' );
```

Comment:
Minor: the spacing for many of the require statements, seems off. Should be:

`require( '' );`